web/src/components/StreamBadge/StreamBadge.scss

```diff
@@ -14,8 +14,10 @@
 }
 
 .stream-badge-name {
-  color: general.$text-color;
+  overflow: hidden;
+  text-overflow: ellipsis;
   white-space: nowrap;
+  color: general.$text-color;
```

Comment:
pls revert and squash this `color` line change if you're gonna keep this line; it's  bad form to screw up `git blame` like this